File: TransCoder_tokenized_test_set_functions/java/NOBLE_INTEGERS_IN_AN_ARRAY_COUNT_OF_GREATER_ELEMENTS_IS_EQUAL_TO_VALUE.java
public static int nobleInteger ( int arr [ ] ) {
  int size = arr . length;
  for ( int i = 0;
  i < size;
  i ++ ) {
    int count = 0;
    for ( int j = 0;
    j < size;
    j ++ ) if ( arr [ i ] < arr [ j ] ) count ++;
    if ( count == arr [ i ] ) return arr [ i ];
  }
  return - 1;
}

File: TransCoder_tokenized_test_set_functions/python/NOBLE_INTEGERS_IN_AN_ARRAY_COUNT_OF_GREATER_ELEMENTS_IS_EQUAL_TO_VALUE.py
def nobleInteger ( arr, size ) :
    for i in range ( 0, size ) :
        count = 0
        for j in range ( 0, size ) :
            if ( arr [ i ] < arr [ j ] ) :
                count += 1
        if ( count == arr [ i ] ) :
            return arr [ i ]
    return - 1
